README-CN.md

```diff
@@ -0,0 +1,203 @@
+# app
+
+[![Build Status](https://travis-ci.org/murlokswarm/app.svg?branch=master)](https://travis-ci.org/murlokswarm/app)
+[![Go Report Card](https://goreportcard.com/badge/github.com/murlokswarm/app)](https://goreportcard.com/report/github.com/murlokswarm/app)
+[![Coverage Status](https://coveralls.io/repos/github/murlokswarm/app/badge.svg?branch=master)](https://coveralls.io/github/murlokswarm/app?branch=master)
+[![awesome-go](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/avelino/awesome-go#readme)
+[![GoDoc](https://godoc.org/github.com/murlokswarm/app?status.svg)](https://godoc.org/github.com/murlokswarm/app)
+
+一个使用
+[Go](https://golang.org), [HTML](https://en.wikipedia.org/wiki/HTML5),
+[CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)构建的多平台UI框架
+
+![ui demo](https://github.com/murlokswarm/app/wiki/assets/ui-demo-large.gif)
+
+[English](./README.md) | 中文
+
+## 目录
+
+* [安装](#install)
+* [支持平台](#support)
+* [示例：Hello world](#hello)
+* [架构](#architecture)
+* [开发工具：Goapp](#goapp)
+* [文档](#doc)
+* [捐赠](#donate)
+
+<a name="install"></a>
+
+## 安装
+
+```sh
+# Install:
+go get -u -v github.com/murlokswarm/app/...
+
+# Update:
+goapp update -v
+```
+
+<a name="support"></a>
+
+## 支持平台
+
+|平台|状态|
+|:-|:-:|
+|[MacOS](https://godoc.org/github.com/murlokswarm/app/drivers/mac#Driver)|✔|
+|[Web](https://godoc.org/github.com/murlokswarm/app/drivers/web#Driver)|✔|
+|Windows|[🔨](https://github.com/murlokswarm/app/issues/141)|
+|Linux|✖|
+
+<a name="hello"></a>
+
+## Hello world
+
+### 创建步骤
+
+```sh
+# Go to your repository:
+cd YOUR_REPO
+
+# Init the repo:
+goapp mac init
+```
+
+### 代码
+
+```go
+// YOUR_REPO/main.go
+
+// Hello compo.
+type Hello struct {
+    Name string
+}
+
+func (h *Hello) Render() string {
+    return `
+<div class="Hello">
+    <h1>
+        Hello
+        {{if .Name}}
+            {{.Name}}
+        {{else}}
+            world
+        {{end}}!
+    </h1>
+    <input value="{{.Name}}" placeholder="Write a name..." onchange="Name" autofocus>
+</div>
+    `
+}
+
+func main() {
+    app.Import(&Hello{})
+
+    // Use mac driver with Hello compo.
+    app.Run(&mac.Driver{
+        URL: "/hello",
+    })
+}
+```
+
+### 构建并运行
+
+```sh
+# Build and run with debug mode:
+goapp mac run -d
+```
+
+查看 [完整示例](https://github.com/murlokswarm/app/tree/master/examples/hello).
+
+<a name="architecture"></a>
+
+## 架构
+
+![ui architecture](https://github.com/murlokswarm/app/wiki/assets/architecture.png)
+
+### 元素
+
+一个 [Elem](https://godoc.org/github.com/murlokswarm/app#Elem) 代表一个UI组件. 部分元素可以
```

Comment:
Look like there is encoding issue here:
<img width="501" alt="screen shot 2018-09-12 at 12 12 24 am" src="https://user-images.githubusercontent.com/2692731/45408215-96cc9a00-b620-11e8-9f13-0192840802b9.png">

Reply:
done